modules/troubleshooting-network-observability-network-flow.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="configure-network-traffic-console_{context}"]
+= Configuring network flow for the network interfaces
```

Comment:
This also lacks context. The text below does not explain how to configure network interfaces but explaining why two concrete interfaces (`br-int` and `br-ex`) don't show flows.

I'd change the title by:
```
= Not seeing flows from neither the `br-int` nor `br-ex` interfaces
```
Then explainin the cause:

> [br-ex and br-int are virtual bridge devices](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/networking_guide/bridge-mappings), so they operate at OSI Layer 2 (e.g. Ethernet level). The eBPF agent works at Layers 3 and 4 (IP and TCP level), so it is expected that traffic passing through br-int and br-ex is captured by the agent when it is processed by other interfaces (e.g. physical host or virtual pod interfaces).
>
> This means that, if you restrict the agent interfaces (using the interfaces or excludeInterfaces properties) to attach only to br-int and/or br-ex, you won't be able to see any flow.

Reply:
Addressed